Wormholy.xcodeproj/project.pbxproj

```diff
@@ -612,7 +612,7 @@
 				DEVELOPMENT_TEAM = "";
 				GCC_C_LANGUAGE_STANDARD = gnu11;
 				INFOPLIST_FILE = WormholyDemo/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```

Comment:
Why the bump?
I mean, for us I don't care bumping that, but for upstream maybe they'd want to keep retrocompat?

Reply:
reverted